library/build.gradle

```diff
@@ -71,7 +77,8 @@ dependencies {
     implementation "androidx.room:room-runtime:$roomVersion"
     kapt "androidx.room:room-compiler:$roomVersion"
 
-    implementation "com.google.code.gson:gson:$gsonVersion"
+    implementation "com.squareup.moshi:moshi:$moshiVersion"
+    implementation "com.squareup.moshi:moshi-kotlin:$moshiVersion"
```

Comment:
This dependency should be replaced with `kapt "com.squareup.moshi:moshi-kotlin-codegen:1.9.2"` to switch to code generation approach I mentioned on another comment.
Here is another duplicate of the documentation: https://github.com/square/moshi#codegen

Reply:
Replaced `moshi-kotlin` with `moshi-kotlin-codegen`.